challenge.rb

```diff
@@ -1,26 +1,30 @@
 
 def capitalize_each_string(input)
-  #implement your solution here
+  result = []
+  input.each_index{ |idx| result.push(input[idx].capitalize)}
+  return result
```

Comment:
[`Enumerable#map`](http://ruby-doc.org/core-2.2.2/Enumerable.html#method-i-map) should help make this more concise.